.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+<!--
+**Thanks for contributing to Fury.**
+
+**If this is your first time opening a PR on fury, you can refer to [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).**
+
+Contribution Checklist
+
+    - The **Apache Fury (incubating)** community has restrictions on the naming of pr titles. You can also find instructions in [CONTRIBUTING.md](https://github.com/apache/incubator-fury/blob/main/CONTRIBUTING.md).
+
+    - Fury does not accept PRs that degrade performance. If the PR you submit will have an impact on performance, please Benchmark it yourself first.
+-->
+
+## What does this PR do?
+
+<!-- Describe the purpose of this PR. -->
+
+
+## Related issues
+
+<!--
+Is there any related issue? Please attach here.
+
+- #xxxx0
+- #xxxx1
+- #xxxx2
+-->
+
+
+## Does this PR introduce _any_ user-facing change?
+
+<!--
+If any user-facing interface changes, please [open an issue](https://github.com/apache/incubator-fury/issues/new/choose) describing the need to do so and update the document if necessary.
+-->
+
+
+## Benchmark
+
+<!--
+**Fury doesn't accept performance degradation**
```

Comment:
Could we remove this line? The community should be open, this should be case by case.

Reply:
Updated.